src/themes/shared/globalCss.ts

```diff
@@ -0,0 +1,21 @@
+import { defineGlobalStyles, SystemStyleObject } from '@chakra-ui/react';
+
+export const globalCss = defineGlobalStyles({
+  '*, *::before, *::after': {
+    boxSizing: 'border-box',
+  },
+
+  '*': {
+    margin: 0,
```

Comment:
Default margins make sense for text documents but not exactly for web applications